packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.less

```diff
@@ -7,7 +7,6 @@
     &-header {
         overflow: hidden;
         outline: none;
-        cursor: pointer;
```

Comment:
Do we still want to show the pointer cursor for when the user wants to navigate directly to a step by clicking on the header?